DirectProgramming/C++SYCL_FPGA/Tutorials/DesignPatterns/explicit_data_movement/README.md

```diff
@@ -56,36 +56,46 @@ You can also find more information about [troubleshooting build errors](/DirectP
 
-Most of the other code sample tutorials in the [FPGA Tutorials](/DirectProgramming/C++SYCL_FPGA/Tutorials) in the one-API-samples GitHub repository use implicit data movement, which is achieved through the `buffer` and `accessor` SYCL constructs. This tutorial demonstrates an alternative coding style using explicit data movement, which is achieved through USM device allocations.
+Most of the other code sample tutorials in the [FPGA Tutorials](/DirectProgramming/C++SYCL_FPGA/Tutorials) in the oneAPI-samples GitHub repository use implicit data movement, which is achieved through the `buffer` and `accessor` SYCL constructs. This tutorial demonstrates an alternative coding style using explicit data movement, which is achieved through USM allocations.
 
-### SYCL USM Device Allocations
+### SYCL USM Allocations
 
-SYCL USM device allocations enable the use of raw *C-style* pointers in SYCL, called *device pointers*. Allocating space in the FPGA Device Memory for device pointers is done explicitly using the `malloc_device` function. Copying data to or from a device pointer must be done explicitly by the programmer, typically using the SYCL `memcpy` function. Additionally, synchronization between kernels accessing the same device pointers must be expressed explicitly by the programmer using either the `wait` function of a SYCL `event` or the `depends_on` signal between events.
+SYCL USM allocations allows users to allocate memory directly accessible by the FPGA.
+There are three types of such allocations:
+- device allocations (through the use of `malloc_device`). This type of allocation matches the example given above
+- host allocations (through the use of `malloc_host`). The pointer given to the FPGA actually lives in the host DDR, but the BSP implements the necessary functionalities to be able to read directly in this memory.
+- shared allocations (through the use of `malloc_shared`). The memory can live on the host or on the FPGA and may be moved at runtime between the host and device depending on the memory access pattern.
 
-In this tutorial, the `SubmitImplicitKernel` function demonstrates the basic SYCL buffer model, while the `SubmitExplicitKernel` function demonstrates how you may use these USM device allocations to perform the same task and manually manage the movement of data.
+SYCL USM allocations enable the use of raw *C-style* pointers in SYCL. 
+Copying data to or from a USM pointer must be done explicitly by the programmer, typically using the SYCL `memcpy` function. 
+Additionally, synchronization between kernels accessing the same device pointers must be expressed explicitly by the programmer using either the `wait` function of a SYCL `event` or the `depends_on` signal between events.
```

Comment:
This might only apply to USM device allocations. For host/shared allocations memcpy is certainly not required.